ioservice/storage_dev.c

```diff
@@ -309,6 +313,10 @@ static void storage_devs_conf_refresh(struct m0_storage_devs *storage_devs,
 	struct m0_conf_sdev   *conf_sdev;
 	int                    rc;
 
+	struct m0_reqh_context *rctx;
+
+	rctx = m0_cs_reqh_context(reqh);
+
```

Comment:
Group the variable declaration together.

And do the initialization at the same time.
```
struct m0_reqh_context *rctx = m0_cs_reqh_context(reqh);
```

Reply:
Done..